packages/salesforcedx-vscode-apex-replay-debugger/package.json

```diff
@@ -218,7 +218,7 @@
       {
         "type": "apex-replay",
         "label": "Apex Replay Debugger",
-        "program": "./dist/apexreplaydebug.js",
+        "program": "./node_modules/@salesforce/salesforcedx-apex-replay-debugger/out/src/adapter/apexReplayDebug.js",
```

Comment:
So it turned out that I had addressed this update in the packaging and just forgot to put this back to the original value.